### PR TITLE
poweroff, halt, reboot: add examples

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

File: pages/linux/poweroff.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
 
 > Shutdown the system.
+> More information: <https://man7.org/linux/man-pages/man8/reboot.8.html>.
 
-- Poweroff the system:
+- Power off the system:
+
+`poweroff`
+
+- Send an ACPI hardware signal to initiate a complete and immediate shutdown:
```
Comment (navarroaxel) [inline pages/linux/poweroff.md:10]: This new example doesn't add something new, only explain how poweroff works. I think you can work over the `--halt` vs `--poweroff` arguments to add something new, or explain `--wtmp-only`. etc.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/linux/poweroff.md (hunk 1 above)
Comment (marchersimon) [reply pages/linux/poweroff.md:10]: But isn't there a difference between `poweroff` and `sudo poweroff`?

Comment (navarroaxel) [reply pages/linux/poweroff.md]: I think the difference is between `halt`, `poweroff` and `shutdown` shutdown sequences.

File: pages/linux/halt.md [hunk 2]
```diff
@@ -1,16 +1,24 @@
 # halt
 
-> Halt, power-off or reboot the machine.
-> More information: <https://www.freedesktop.org/software/systemd/man/halt.html>.
+> Halt the system.
+> More information: <https://man7.org/linux/man-pages/man8/reboot.8.html>.
```
Comment (navarroaxel) [inline pages/linux/halt.md:4]: Why reboot here?
```suggestion
> More information: <https://www.man7.org/linux/man-pages/man8/halt.8.html>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
could you add the more info link to the `shutdown` page? https://www.man7.org/linux/man-pages/man8/shutdown.8.html

**Comment by bl-ue** [comment]
Based on the description in the man pages, these 3 commands appear to all be identical, and thus it would seem that they ought to be aliased to one another.

From the man pages:
> halt, poweroff, reboot may be used to halt, power-off, or reboot the machine. All three commands take the same options.

**Comment by navarroaxel** [comment]
> Based on the description in the man pages, these 3 commands appear to all be identical, and thus it would seem that they ought to be aliased to one another.

@bl-ue but they have differences:
https://askubuntu.com/questions/64995/what-is-the-difference-between-shutdown-and-poweroff-commands#64999
https://www.tecmint.com/shutdown-poweroff-halt-and-reboot-commands-in-linux/

**Comment by bl-ue** [comment]
Ah, okay, nevermind then.

**Review by sbrl — APPROVED** [review]
Thanks, @marchersimon!

File: pages/linux/halt.md (hunk 2 above)
Comment (marchersimon) [reply pages/linux/halt.md:4]: Because that url covers all three commands and I didn't know there was a url for reboot too

**Review by navarroaxel — APPROVED** [review]

**Comment by sbrl** [comment]
Why the confused emoji, @navarroaxel? I might have missed something here?

**Comment by navarroaxel** [comment]
@sbrl Because I didn't understand your comment, but it was just a bug in the GitHub app for Android :joy: 
I wondered why you quoted the answer from @bl-ue.
https://user-images.githubusercontent.com/1536740/112724431-12964580-8ef2-11eb-91aa-567d3c89de68.mp4

**Pull request merged**